aliases/available/dnf.aliases.bash

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#zuck007
+cite 'about-alias'
+about-alias 'dnf aliases for fedora 22+ distros'
+
+#set-dnf-aliases
+function _set_dnf_aliases() 
```

Comment:
Why this function? Any reason why this is not just an `if` statement? The function doesn't really serve any function beyond the definition of the aliases as far as I can see...